src/EA.Weee.Web/Areas/Scheme/Views/TransferEvidence/_ViewTransferEvidenceNotePartial.cshtml

```diff
@@ -86,28 +86,42 @@
 <!-- screen 6.07a -->
 
 
-<div class="govuk-width-container govuk-!-padding-top-3 govuk-!-padding-left-0 printable-padding-top-3">
-    <div class="govuk-grid-column-one-third govuk-!-padding-left-0">
-        <span id="site" class="govuk-label--s">Transferred by</span><br />
-        <p aria-labelledby="site" class="govuk-body weee-word-break"> @Html.Raw(Model.TransferredByAddress)</p>
-    </div>
-    <div class="govuk-grid-column-one-third govuk-!-padding-left-0">
-        <span id="recipient" class="govuk-label--s">Recipient</span><br />
-        <p aria-labelledby="recipient" class="govuk-body weee-word-break">@Html.Raw(Model.RecipientAddress)</p>
-    </div>
+    <div class="govuk-width-container govuk-!-padding-top-3 govuk-!-padding-left-0 printable-padding-top-3">
 
+        @if (Model.Status.Equals(NoteStatus.Approved))
+        {
```

Comment:
@GuidoWorkDev  Undo these changes in the view we shouldnt have any changes in here